projects/core/src/cms/config/cms-config.ts

```diff
@@ -22,6 +22,7 @@ export interface StandardCmsComponentConfig {
   ProductReferencesComponent?: CmsComponentMapping;
   CMSTabParagraphComponent?: CmsComponentMapping;
   LoginComponent?: CmsComponentMapping;
+  ScrollToTopComponent?: CmsComponentMapping;
```

Comment:
Just for the sake of discussion, could we have used a CMSFlexComponent?

Reply:
Not sure, let's discuss next huddle